pages/attributes.vue

```diff
@@ -0,0 +1,342 @@
+<template>
+  <v-layout column>
+    <h1 class="display-4 ma-5">Attributes</h1>
+
+    <p :class="paragraph">
+      In this guide, you will learn fundamental elements to understand and 
+      manipulate attributes in OpenGeode.
+    </p>
+
+    <h2 :class="section">What are attributes?</h2>
+
+    <p
+      :class="paragraph"
+    >Attributes stand for properties you can add on OpenGeode objects, typically meshes. 
+    These properties are not directly stored on meshes, so you can imagine add Attributes
+    on other objects. </p>
+
+    <p
+      :class="paragraph"
+    >Attributes are managed by an 
+    <span v-html="attributeManager" />.
```

Comment:
attributeManager or AttributeManager ? In all the file

Reply:
It appears as "AttributeManager" and link to the right page